src/Views/AppInfoView.vala

```diff
@@ -211,6 +226,27 @@ namespace AppCenter.Views {
             });
         }
 
+        private void reload_css () {
+            var provider = new Gtk.CssProvider ();
+            try {
+                string color_primary = package.get_color_primary ();
+                if (color_primary == null) {
+                    color_primary = "rgba(66,83,129,255)";
```

Comment:
since this is 255, is there a reason to use RGBA instead of hex? Probably just #68758e is good